app/src/main/java/com/sdsmdg/pulkit/callingtext/BaseActivity.java

```diff
@@ -106,11 +106,14 @@ protected void onCreate(Bundle savedInstanceState) {
     public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
         super.onRequestPermissionsResult(requestCode, permissions, grantResults);
 
+        Log.d("lololo", "took permissions");

```

Comment:
Define and use a proper LOG_TAG here. See this https://developer.android.com/reference/android/util/Log.html